n2survey/lime/survey.py

```diff
@@ -96,22 +97,37 @@ class LimeSurvey:
 
     def __init__(
         self,
-        structure_file: str,
```

Comment:
This will break existing scripts. Instead, let's keep this argument but make it optional

```
structure_file: Optional[str] = None,
```